src/Mqtt/MqttProcessor.cpp

```diff
@@ -19,6 +19,8 @@
 #include "FotaEvent.h"
 #include "Logger.h"
 
+#include "nlohmann/json.hpp"
```

Comment:
Optional: just an idea - maybe it would make sense not to use the json parser directly here, in the business level part, and to move the processing to the MqttProcessor? so that the library dependency is not introduced on this level.

Reply:
Thank you! Good point. Without significant rework of transport layer it is not possible because at least all json exceptions have to be wrapped and translated into normal std::exception.